### ISSUE TITLE
LevelLoader fails when "x" or "y" not specified for platform

### ISSUE THREAD

**Issue by jarrodparkes** [udacity/ud406#11]
I was creating my own level using Overlap2D and I placed a platform at position (0, 0). Oddly, this caused the .dt file (json) to remove the key/value pairs for "x" and "y" from that particular platform. Thus, when the LevelLoader encountered that platform it crashed because it does not check for the null case.

Here's a quick fix I wrote in LevelLoader.java's loadPlatform method (I should do a PR, but it is late and I'm lazy :disappointed: ):

```java
Number x = ((Number) platformObject.get(Constants.LEVEL_X_KEY));
Number y = ((Number) platformObject.get(Constants.LEVEL_Y_KEY));

final float finalX = (x == null) ? 0 : x.floatValue();
final float finalY = (y == null) ? 0 : y.floatValue();

final float width = ((Number) platformObject.get(Constants.LEVEL_WIDTH_KEY)).floatValue();
final float height = ((Number) platformObject.get(Constants.LEVEL_HEIGHT_KEY)).floatValue();

final Platform platform = new Platform(finalX, finalY + height, width, height);
```

**Comment by JeremySilverTongue** [comment]
Roger. Got a fix. Will be putting into lesson 7 soon. If only I had all the git history ironed out, so I could fix this with a single rebase :P

**Comment by JeremySilverTongue** [comment]
Fixed in 2.7 by https://github.com/udacity/ud406/commit/28a79215ea2da9f968f6709bd210fde5a8fc1ee8

Will be fixed in 2.8 as well. Thanks for the report.
